src/state/contexts/APIContext.js

```diff
@@ -155,19 +158,28 @@ const APIProvider = ({ children }) => {
   const getCustomerByID = authState => {
     const headers = getAuthHeader(authState);
 
-    return axios
-      .get(`${process.env.REACT_APP_API_URI}/customers/${userInfo.sub}`, {
-        headers,
-      })
-      .then(res => {
-        if (res.data) {
-          setCustInfo(res.data);
-          setIsRegistered(true);
-        }
-      })
-      .catch(err => {
-        console.log(err);
-      });
+    return (
+      axios
+        .get(`${process.env.REACT_APP_API_URI}/customers/${userInfo.sub}`, {
+          headers,
+        })
+        .then(console.log('userInfo.sub: ', userInfo.sub)) // this seems to be what's getting lost and crashing the customer dashboard
+        .then(res => {
+          if (res.data) {
+            setCustInfo(res.data);
+            setIsRegistered(true);
+          }
+        })
+        // .get(`${process.env.REACT_APP_API_URI}/customers/${userInfo.sub}`, {
```

Comment:
does this code need to be there? If not, I would just delete it before we merge. Otherwise make sure to leave a comment as to what this code is used for.

Reply:
This was part of a test call I was using when I was tracing the source of the customer dashboard refresh crash. I removed it, because I think we know part of the cause now. It happens when userInfo.sub is lost from state. The place where I think it happens is still indicated with a comment in this file, on line 165.